tabbycat/privateurls/utils.py

```diff
@@ -13,20 +15,21 @@ def generate_url_key(length=8):
     return ''.join(SystemRandom().choice(chars) for _ in range(length))
 
 
-def populate_url_keys(queryset, length=8, num_attempts=10):
+def populate_url_keys(people, length=8, num_attempts=10):
     """Populates the URL key field for every instance in the given QuerySet."""
-    for instance in queryset:
+    existing_keys = list(Person.objects.exclude(url_key__isnull=True).values_list('url_key', flat=True))
```

Comment:
Not using a filter to exclude people of other tournaments, but it should still be fine.